test_runner/performance/test_branching.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import Path
+import timeit
+from typing import List
+from fixtures.benchmark_fixture import PgBenchRunResult
+from fixtures.compare_fixtures import NeonCompare
+from performance.test_perf_pgbench import utc_now_timestamp
+
+
+def test_compare_child_and_root_pgbench_perf(neon_compare: NeonCompare):
```

Comment:
Add comment. What does this test? What numbers does it output and how to interpret them?

Same with the tests after this. Or maybe add one comment to the top of the file to cover all the tests in the file, they seem similar to each other.